decidim-proposals/app/helpers/decidim/proposals/application_helper.rb

```diff
@@ -139,18 +139,10 @@ def vote_button_for(model, from_proposals_list)
         render partial: "decidim/proposals/proposals/participatory_texts/proposal_vote_button.html", locals: { proposal: model, from_proposals_list: from_proposals_list }
       end
 
-      def endorsers_for(proposal)
-        proposal.endorsements.for_listing.map { |identity| present(identity.normalized_author) }
-      end
-
       def form_has_address?
         @form.address.present? || @form.has_address
       end
 
-      def authors_for(collaborative_draft)
-        collaborative_draft.identities.map { |identity| present(identity) }
-      end
-
```

Comment:
can you confirm that this two methods are not used? (I can't see you are extracting them to some other place..)

Reply:
Confirmed, they aren't used!